apps/src/lab2/projects/utils.ts

```diff
@@ -21,9 +24,24 @@ export function getStandaloneProjectId(): string | undefined {
   return appOptions.channel;
 }
 
-export function getProjectType(): string {
+export function getServerLevelId(): number | undefined {
   const appOptions = getScriptData('appoptions') as PartialAppOptions;
-  return appOptions.projectType;
+  return appOptions.levelId;
+}
+
+export function isShareView(): boolean | undefined {
+  const appOptions = getScriptData('appoptions') as PartialAppOptions;
+  return appOptions.share;
+}
+
+export function isIframeEmbed(): boolean | undefined {
+  const appOptions = getScriptData('appoptions') as PartialAppOptions;
+  return appOptions.iframeEmbed;
+}
+
+export function isIframeEmbedAppAndCode(): boolean | undefined {
+  const appOptions = getScriptData('appoptions') as PartialAppOptions;
+  return appOptions.iframeEmbedAppAndCode;
 }
```

Comment:
Not very familiar with all of these.  Do we want to support all of them in Lab2?

Reply:
I included them since we eventually may want to? iframeEmbed is the same as the share view, just embedded in an iframe. iframeEmbedAppAndCode is supposed to present a readonly view of the full workspace in an iframe

Reply:
Over the Lab2 process I think we've found value in reducing our surface area to key features we want to support going forward, so I'd encourage just supporting must-have scenarios to begin with.  Though maybe add a comment alluding to additional scenarios we could re-introduce if there is demand.